app/views/templates/_help_accordion.html.erb

```diff
@@ -1,12 +1,12 @@
 <div class="panel panel-default">
   <div class="panel-heading">
     <h4 class="panel-title">
-      <a data-toggle="collapse" data-parent="#accordion-markup" href="#<%= title.parameterize %>" class="collapsed">
-        <%= title %>
+      <a data-toggle="collapse" data-parent="#accordion-markup" href="#<%= _(title.parameterize) || title.parameterize %>" class="collapsed">
```

Comment:
Isn't a cleaner solution to use something else than titles in a href? Titles are human readable, but that also breaks things when you switch the language. IMHO they should at least be untranslated strings so the application behaves the same way no matter the language chosen.

An alternative is to use some real ID. That's probably more predictable.

Reply:
This is a small helper so we can keep using titles, but untranslated. Thanks for the feedback!